Content.Server/GameTicking/Rules/TraitorRuleSystem.cs

```diff
@@ -58,6 +61,7 @@ private void AfterEntitySelected(Entity<TraitorRuleComponent> ent, ref AfterAnta
     private void SetCodewords(TraitorRuleComponent component)
     {
         component.Codewords = GenerateTraitorCodewords(component);
+        _adminLogger.Add(LogType.EventStarted, LogImpact.Low, $"Codewords generated: {string.Join(", ", component.Codewords)}");
```

Comment:
should log the rule too since multiple rules can have codewords

Reply:
Very good point, fixed it now

Just wanna say thanks for reviewing a small little admin PR like this btw, you're awesome